pr_agent/git_providers/gitlab_provider.py

```diff
@@ -313,10 +341,20 @@ def get_repo_settings(self):
             return ""
 
     def add_eyes_reaction(self, issue_comment_id: int) -> Optional[int]:
-        return True
+        try:
+            reaction = self.pr.get_issue_comment(issue_comment_id).create_reaction("eyes")
+            return reaction.id
+        except Exception as e:
+            logging.exception(f"Failed to add eyes reaction, error: {e}")
+            return None
 
     def remove_reaction(self, issue_comment_id: int, reaction_id: int) -> bool:
```

Comment:
**Suggestion:** Proposed documentation
```suggestion
    def remove_reaction(self, issue_comment_id: int, reaction_id: int) -> bool:
    """
    This method removes a specific reaction from a specific issue comment. It returns True if successful, 
    otherwise it logs the exception and returns False.
    """
```